tests/Renderer/QueryResultsRenderer.test.ts

```diff
@@ -61,7 +62,20 @@ describe('QueryResultsRenderer - accessing results', () => {
         await renderer.render(State.Warm, aTask, document.createElement('div'));
 
         expect(renderer.queryResult.totalTasksCount).toEqual(1);
-        expect(renderer.filteredQueryResult.totalTasksCount).toEqual(1);
+    });
+
+    it.skip('should have actual result after filtering results', async () => {
```

Comment:
For future reference, `it.skip()` is really worth avoiding, and `it.failing()` is much better to use.

- `it.skip()` just never runs the test, so you don't ever get any feedback from it.
- `it.failing()` requires that the test will fail - and itself fails if all parts of the test fail.
  - This means that previously-failing tests that get fixed in future will be discovered, marked as no-long-failing, and will continue to test the new behaviour...